docs/reference/api/docker_remote_api_v1.23.md

```diff
@@ -2830,7 +2830,10 @@ Status Codes:
 
 Query Parameters:
 
-- **filters** - JSON encoded value of the filters (a `map[string][]string`) to process on the volumes list. There is one available filter: `dangling=true`
+- **filters** - JSON encoded value of the filters (a `map[string][]string`) to process on the volumes list. The filter value is one of:
```

Comment:
I'd change `"The filter value is one of:"` to `"Available filters:"`, otherwise it may be confusing, because each filter is a name/value pair.